EidSuggest/Suggest.php

```diff
@@ -25,43 +25,37 @@
 
 # TSFE initialization
 
-tslib_eidtools::connectDB();
-$pageId     = filter_var(t3lib_div::_GET('id'), FILTER_SANITIZE_NUMBER_INT);
+$pageId     = filter_var(\TYPO3\CMS\Core\Utility\GeneralUtility::_GET('id'), FILTER_SANITIZE_NUMBER_INT);
 $languageId = filter_var(
-	t3lib_div::_GET('L'),
+	\TYPO3\CMS\Core\Utility\GeneralUtility::_GET('L'),
 	FILTER_VALIDATE_INT,
 	array('options' => array('default' => 0, 'min_range' => 0))
 );
-
-$TSFE = t3lib_div::makeInstance('tslib_fe', $GLOBALS['TYPO3_CONF_VARS'], $pageId, 0, TRUE);
-$TSFE->initFEuser();
-$TSFE->initUserGroups();
+$GLOBALS['TSFE'] = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('TYPO3\\CMS\\Frontend\\Controller\\TypoScriptFrontendController', $GLOBALS['TYPO3_CONF_VARS'], $pageId, 0, TRUE);
+$GLOBALS['TSFE']->initFEuser();
+$GLOBALS['TSFE']->initUserGroups();
 // load TCA
-if (version_compare(TYPO3_version, '6.1-dev', '>=')) {
-	\TYPO3\CMS\Core\Core\Bootstrap::getInstance()->loadCachedTca();
-} else {
-	$TSFE->includeTCA();
-}
-$TSFE->sys_page = t3lib_div::makeInstance('t3lib_pageSelect');
-$TSFE->rootLine = $TSFE->sys_page->getRootLine($pageId, '');
-$TSFE->initTemplate();
-$TSFE->getConfigArray();
+\TYPO3\CMS\Frontend\Utility\EidUtility::initTCA();
```

Comment:
Yes initTca is 6.2 compatible:
https://git.typo3.org/Packages/TYPO3.CMS.git/blob/f695f377f48c23bec3156c6d0f58f8a078d8a651:/typo3/sysext/frontend/Classes/Utility/EidUtility.php#l78

The $TSFE is not exactly removed but was replaced with $GLOBALS['TSFE'] instead.